src/Libraries/Microsoft.Extensions.Diagnostics.Probes/TcpEndpointHealthCheckExtensions.cs

```diff
@@ -12,17 +12,17 @@
 namespace Microsoft.Extensions.Diagnostics.Probes;
```

Comment:
Change to Microsoft.Extensions.DependencyInjection, that's where extensions on IServiceCollection go.